source/linux/How_to_Guides/Target/How_To_Enable_M2CC3301_in_linux.rst

```diff
@@ -74,12 +81,23 @@ Connect to Wi-Fi
 Using scripts provided in the SDK makes connecting to an Access Point or router straightforward.
 The following are steps to connect to a WPA password-secured Access Point. 
 
-.. code-block:: console
+.. ifconfig:: CONFIG_sdk in ('SITARA')
+
+    .. code-block:: console
+
+        cd /usr/share/cc33xx
+        ./sta_start.sh
+        ./sta_connect.sh -s WPA-PSK -n <SSID> -p <PASSWORD>
+        udhcpc -i wlan0
+
+.. ifconfig:: CONFIG_sdk in ('DebianSDK')
+
+    .. code-block:: console
 
-    cd /usr/share/cc33xx
-    ./sta_start.sh
-    ./sta_connect.sh -s WPA-PSK -n <SSID> -p <PASSWORD>
-    udhcpc -i wlan0
+        cd /usr/share/cc33xx
+        bash ./sta_start.sh
+        bash ./sta_connect.sh -s WPA-PSK -n <SSID> -p <PASSWORD>
+        udhcpc -i wlan0
```

Comment:
Can you split this change into a separate commit so it can be reverted when the upstream issue is addressed?

Reply:
Done, @StaticRocket